app/controllers/idv/in_person/address_controller.rb

```diff
@@ -121,6 +122,10 @@ def confirm_in_person_address_step_needed
         return if request.referer == idv_in_person_verify_info_url
         redirect_to idv_in_person_ssn_url
       end
+
+      def set_usps_form_presenter
+        @presenter = Idv::InPerson::UspsFormPresenter.new
+      end
```

Comment:
Can you add a spec for this controller?

Reply:
Yup sure thing. Assuming I should just follow the same conventions as other controllers for the filename?